examples/plot_h2_square.py

```diff
@@ -0,0 +1,39 @@
+"""
+Plot a square on a h2 space,
+with Poincare Disk visualization.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from geomstats.hyperbolic_space import HyperbolicSpace
+import geomstats.visualization as visualization
+
+H2 = HyperbolicSpace(dimension=2)
+METRIC = H2.metric
+SQUARE_SIZE = 5.99
+
+
+def main():
+    top = SQUARE_SIZE / 2.0
+    bot = - SQUARE_SIZE / 2.0
+    left = - SQUARE_SIZE / 2.0
+    right = SQUARE_SIZE / 2.0
+    corners_int = [(bot, left), (bot, right), (top, right), (top, left)]
+    corners_ext = H2.intrinsic_to_extrinsic_coords(np.array(corners_int))
```

Comment:
No way you can the explicit conversion to np.array, using the new vectorization.to_ndarray function for example?